exercises/concept/documented-lasagna/runtests.jl

```diff
@@ -0,0 +1,30 @@
+using Test
+
+include("lasagna.jl")
+
+# Util function
+function hasdocstring(f::Symbol)
+    @eval !startswith(string(@doc $f), "No documentation found.")
```

Comment:
I've asked on [Zulip](https://julialang.zulipchat.com/#narrow/stream/225542-helpdesk/topic/finding.20out.20if.20a.20function.20has.20a.20docstring/near/231159423) if there's a better way. I couldn't find an internal function for it. Do you know one?

Reply:
Seb's answer with Docs.meta seems good